modules/systemtests/src-test/org/ejbca/core/protocol/cmp/CmpTestCase.java

```diff
@@ -308,8 +309,14 @@ private void cleanup() throws AuthorizationDeniedException {
     }
 
     protected final void removeCAs(CA[] cas) throws AuthorizationDeniedException {
+        final InternalCertificateStoreSessionRemote internalCertificateStoreSession = EjbRemoteHelper.INSTANCE.getRemoteSession(
+                InternalCertificateStoreSessionRemote.class, EjbRemoteHelper.MODULE_TEST);
         for (CA ca : cas) {
             if (ca != null ) {
+                
+                // Delete certs issued by the CA
+                internalCertificateStoreSession.removeCertificatesByIssuer( ca.getSubjectDN());
```

Comment:
This is done inside CaTestUtils with the addition in that file. Not needed here.